apps/search/js/providers/suggestions.js

```diff
@@ -44,8 +52,6 @@
 
     name: 'Suggestions',
 
-    remote: true,
```

Comment:
The pull request in the current state seems to work, but I don't really like the removal of this. It kind of loses its meaning if we're not using it as before. It's also less consistent if not all remote providers have this.

Additionally it seems we're falling for the infamous "boolean trap" by adding this boolean argument.  Can we instead just read from the global Search object, and access Search.suggestionsEnabled? We already rely on the Search object in this file, so I believe this is cleaner.